### ISSUE TITLE
Hidpi (Retina) support

### ISSUE THREAD

**Issue by probonopd** [furybsd/furybsd-livecd#183]
Hidpi (Retina) is not detected out of the box in the KDE Live systems (I did not try the other ones yet).
Can we make it work?

**Comment by pkgdemon** [comment]
Yes.  It's normally specific per DE.  I think this is a source of some good information I have used in the past for various desktops.  https://wiki.archlinux.org/index.php/HiDPI
